### PR TITLE
sandbox: move proxy plugin

### PR DESCRIPTION
**Pull request by mfrancisc**
Move proxy plugin configuration from sandbox-sre

### CODE REVIEW

File: components/sandbox/toolchain-host-operator/base/proxy/proxyplugin.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: toolchain.dev.openshift.com/v1alpha1
+kind: ProxyPlugin
+metadata:
+  name: tekton-results
+  namespace: toolchain-host-operator
+spec:
+  openShiftRouteTargetEndpoint:
+    # see https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/tekton-results/base/api-route.yaml
```
Comment (sadlerap) [inline components/sandbox/toolchain-host-operator/base/proxy/proxyplugin.yaml:8]: This URL doesn't resolve, did you mean this instead?
```suggestion
    # see https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/gitops/argocd/pipeline-service/tekton-results/api-route.yaml
```

**Review by filariow — APPROVED** [review]

**Pull request merged**